chains/ethereum/config.go

```diff
@@ -18,11 +19,12 @@ const DefaultGasPrice = 20000000000
 
 // Config encapsulates all necessary parameters in ethereum compatible forms
 type Config struct {
-	name         string      // Human-readable chain name
-	id           msg.ChainId // ChainID
-	endpoint     string      // url for rpc endpoint
-	from         string      // address of key to use
-	keystorePath string      // Location of keyfiles
+	name         string       // Human-readable chain name
+	id           msg.ChainId  // ChainID
+	endpoint     string       // url for rpc endpoint
+	from         string       // address of key to use
+	keystorePath string       // Location of keyfiles
+	chainLog     log15.Logger // Error logger for the chain
```

Comment:
this should just be called `logger`

Reply:
This shouldn't be here, it should be passed directly into Listener, Writer and Connection.

Please use the name `log` throughout, the fewer characters the better,